src/main/java/com/example/java/maven/cardGamePeterPan/Deck.java

```diff
@@ -0,0 +1,38 @@
+package com.example.java.maven.cardGamePeterPan;
+
+import java.util.Collections;
+import java.util.Stack;
+
+public class Deck {
+    private final Stack<Card> cards;
+
+    public Stack<Card> getCards() {
+        Stack<Card> cardsCopy = new Stack<>();
+        cardsCopy.addAll(cards);
+        return cardsCopy;
+    }
+
+    public Deck() {
+        this.cards = new Stack<>();
+        for (Card.Suit suit : Card.Suit.values()) {
+            if (suit != Card.Suit.WITCH)
```

Comment:
rather than these ifs I would prefer having methods in `Suit` and `Rank`: `getValuesWithoutWitch()`

Reply:
Created these methods and simplified creating deck